_studio/shared/include/mfx_config.h

```diff
@@ -130,16 +96,12 @@
     #if defined(AS_H264LA_PLUGIN)
         #undef MFX_ENABLE_MJPEG_VIDEO_DECODE
         #undef MFX_ENABLE_MJPEG_VIDEO_ENCODE
-        #undef MFX_ENABLE_H264_VIDEO_FEI_ENCPAK
-        #undef MFX_ENABLE_H264_VIDEO_FEI_PREENC
-        #undef MFX_ENABLE_H264_VIDEO_FEI_ENC
-        #undef MFX_ENABLE_H264_VIDEO_FEI_PAK
+        #undef MFX_ENABLE_H264_VIDEO_FEI_ENCODE
         #undef MFX_ENABLE_VPP
     #endif
 
-    #if defined(AS_HEVCD_PLUGIN) || defined(AS_HEVCE_PLUGIN) || defined(AS_VP8D_PLUGIN) || defined(AS_VP9D_PLUGIN) || (defined(AS_HEVC_FEI_ENCODE_PLUGIN) && MFX_VERSION >= MFX_VERSION_NEXT)
+    #if defined(AS_HEVC_FEI_ENCODE_PLUGIN)
```

Comment:
No, it should not because we already cover HEVC FEI CMakeLists.txt script with the same version check:
https://github.com/Intel-Media-SDK/MediaSDK/blob/1ff19baf50063e3be25837c3fd8202dfa4ee0d36/_studio/hevc_fei/CMakeLists.txt#L21
We simply will not build it if version is wrong, thus no need to check version here. Moreover, AS_HEVC_FEI_ENCODE_PLUGIN is defined in the above cmake file, thus, it is already under version check.